frontend/src/components/layout.ts

```
@@ -208,80 +204,105 @@ type NModalProps = {
+      {id: "negative", text: text.negative},
     ],
-    on: {close: (id: "positive" | "negative") => {
+    on: {close: (id?: "positive" | "negative") => {
+      if (id === undefined)
+        return on.close();
+
       switch (id) {
-        case "positive": return props.on.positive_click();
-        case "negative": return props.on.negative_click();
-        default: return props.on.close();
+        case "positive": on.positive_click();
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)